.pre-commit-config.yaml

```diff
@@ -71,8 +71,39 @@ repos:
   rev: pylint-2.5.0
   hooks:
     - id: pylint
+      language_version: python3
+      args:
+      - --disable=import-error # E0401. Locally you could not have all imports.
+      - --disable=fixme # W0511. 'TODO' notations.
```

Comment:
It's currently recommended to generate a config with smth like `pylint --rcfile > .pylintrc`, change and commit it. This allows to see what all options are, their default values and it allows other tools/editors to reuse it.